app/graphql/input_types/model_attributes.rb

```diff
@@ -0,0 +1,10 @@
+module InputTypes
+  class ModelAttributes < InputTypes::BaseInputObject
```

Comment:
This is unused currently. I guess you wanted to reuse that in create and update mutations. I tried to do that by myself, however I wasn't able to find correct way of reusing it in mutations.